### ISSUE TITLE
relationship between superpoint_v1.pth and superpointbn.pth

### ISSUE THREAD

**Issue by fish258** [shaofengzeng/SuperPoint-Pytorch#7]
I guess superpoint_v1 is the pytorch version model file converted by the TF version. Is the superpointbn.pth have same weights as the former one except the modified parts (I see that in the bn version, you made some modifications on the architecture)?

**Comment by shaofengzeng** [comment]
Hi, superpoint_v1.pth is the weight released by magicleap. superpoint_bn.pth is the weight released by rpautrat's superpoint. I have changed their architectures so that you can easily load them.
For superpoint_bn.pth, since the `conv2d`, `batch normalization`  are different between tf and pytorch, so you can only achieve similar performance.
